solr/solr-ref-guide/src/index.adoc

```diff
@@ -10,6 +10,7 @@
     streaming-expressions, \
     solrcloud, \
     legacy-scaling-and-distribution, \
+    circuit-breakers, \
```

Comment:
FYI @ctargett  I'm not sure if this page deserves its own page or should be a part of another... and if it is its own page as Atris did then where the logical placement is.  Any way; I just want to bring this to your attention.

Reply:
I missed this before it was committed - it shouldn't be a top-level page IMO. I'm not sure where to put it, and I'm also working on a major re-org of all pages, so I'll leave it for now and figure out where it should go as part of the re-org process.